src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -9,6 +9,7 @@
   --pf-c-dropdown__toggle--MinWidth: var(--pf-global--target-size--MinWidth);
```

Comment:
can you also add `--pf-c-dropdown__toggle-button: var(--pf-global--color--100);` and then `color: var(--pf-c-dropdown__toggle-button);` to `pf-c-dropdown__toggle-button`? You can verify this fixes the split button dropdown in a dark theme. Currently the toggle button is black in a dark theme and it should turn white.

Reply:
Yes working now!